block/cardano-sl-block.cabal

```diff
@@ -13,7 +13,6 @@ extra-source-files:  README.md
 cabal-version:       >=1.10
 
 library
-  default-language: Haskell2010
```

Comment:
@parsonsmatt You don't like stuff older than 8 years? :trollface:

Reply:
It's duplicated in the cabal file :o